app/models/item.rb

```diff
@@ -15,8 +18,19 @@ def self.sql_injection(params)
     attributes = Item.first.attributes if Item.first
     search_terms = params.permit(attributes.keys).to_h
     injection = search_terms.map do |k,v|
-      "lower(#{k}) LIKE '%#{v.downcase}%'"
+      param_generator(k, v)
+      # "lower(#{key}) LIKE '%#{value.downcase}%'"
     end
-    injection.join(" OR ")
+    injection.join(" AND ")
   end
+
+  def self.param_generator(key, value)
+    if key.include?('ated_at')
+      value = value.to_datetime
```

Comment:
Change to datetime, otherwise it will break

Reply:
Then compare the current value to itself (WHICH SHOULD BE ENOUGH) and a day ahead